File: yl2.py
import math

print("Ringi pindala ja ümbermõõt")
radius = float(input("Sisesta raadius: "))

print(f"pindala:\t{radius * math.pi ** 2:.4f}")
print(f"ümbermõõt:\t{2 * radius * math.pi:.4f}")

File: yl2.lua
--------------------------------------------------------------------------------
-- circle formulas
--------------------------------------------------------------------------------

io.write [[
Ringi pindala ja ümbermõõt
Sisesta raadius: ]]

local radius = tonumber(io.read("*line"))

print(("pindala:\t%.4f"  ):format(radius * math.pi ^ 2))
print(("ümbermõõt:\t%.4f"):format(2 * radius * math.pi))


--------------------------------------------------------------------------------
